wolfssl/internal.h

```diff
@@ -1227,7 +1227,7 @@ enum {
         #error "MySQL needs SP_INT_BITS at least at 8192"
     #endif
 
-    #if WOLFSSL_MAX_RSA_BITS > ENCRYPT_BASE_BITS
+    #if WOLFSSL_MAX_RSA_BITS > SP_INT_BITS
```

Comment:
Can't change this. It must be generic across maths implementations.

Reply:
That clause is inside your ```#elif defined(WOLFSSL_SP_MATH_ALL) || defined(WOLFSSL_SP_MATH)``` section.  Why does it need to be generic?

More generally, why are you asserting `ENCRYPT_BASE_BITS <= WOLFSSL_MAX_RSA_BITS`, while putting an 8192 bit ceiling on `ENCRYPT_BASE_BITS`?  That new code makes it impossible to configure a TLS build for max RSA keysize > 8192 bits.  If I understand correctl, this is a frivolous assert.  In any case, it's breaking `wolfssl-multi-test.sh all-crypto-linuxkm-defaults-max-func-stack-2k-build`, which is our test that screens for public keys errantly placed on the stack in a smallstack build.  That test scenario uses this config:

```--enable-linuxkm-defaults --enable-all --enable-crypttests --disable-benchmark --disable-examples --enable-aesni --enable-sp-asm --with-max-rsa-bits=16384 CFLAGS="-Wframe-larger-than=2048 -Wstack-usage=4096"```

FWIW this build will still succeed on master if `--enable-cryptonly` is added, but we need to screen for PK objects on the stack in the TLS smallstack build.